recipes/libcufft/meta.yaml

```diff
@@ -0,0 +1,110 @@
+{% set name = "cufft" %}
+{% set version = "11.0.0.21" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/libcufft/{{ platform }}/libcufft-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 4650f79fcf377af410afedb56445f1332e8f213ec3c94d660f9a91328c3ecb4a  # [linux64]
+  sha256: e711671995e9d92d16015a4cd3751ec30710a49fa22f2b88a4a42905c248d861  # [ppc64le]
+  sha256: 91aa1ba5e68ca65778222c9a427a9214c4550d2f45e978e51779a207bafc17a3  # [aarch64]
+  sha256: a33f577d48ef432d5cfa8d0ebfa789cc5261af5dd84fd9c608235edc3f769e43  # [win]
+
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:          # [linux]
+    strong:             # [linux]
+      - __glibc >=2.17  # [linux]
+  missing_dso_whitelist:
+     - /lib64/libdl.so.2            # [linux]
+     - /lib64/libm.so.6             # [linux]
+     - /lib64/libpthread.so.0       # [linux]
+     - /lib64/librt.so.1            # [linux]
+     - /lib64/libgcc_s.so.1         # [linux]
+     - /lib64/libc.so.6             # [linux]
+     - /lib64/ld-linux-x86-64.so.2  # [linux]
```

Comment:
Think we should be able to drop these now that we are requiring the `compiler`s below

```suggestion
```

Reply:
Hmm, hopefully it works, I had spend quite some time trying to understand why it might be failing (locally mostly) and not making progress.

Reply:
@jakirkham it is now running into requiring the list and I have no idea how to understand it, it seems to me sysroot should have this and that it gets installed?  On the cublas one it seems to not require it (except for the last build where the library isn't needed to be included).